src/include/trn_datamodel.h

```diff
@@ -32,6 +32,9 @@
 #define TRAN_MAX_NSWITCH 128
 #define TRAN_MAX_NROUTER 128
 #define TRAN_MAX_REMOTES 128
+#define TRAN_MAX_NSWITCH_OFFLOAD 5
```

Comment:
What's the reason for these limits?

Reply:
The size of key+value has a limitation (\i.e., 64B) when the map is offloaded to Netronome Smart NIC.
Thus, in the offloaded map named network_t_offload, the number of switches_ips must be limited and we set the number as 5.